autotest.pm

```diff
@@ -275,8 +275,10 @@ sub run_all () {
     }
     eval {
         bmwqemu::save_vars(no_secret => 1);
+        bmwqemu::diag("Sending tests_done");
```

Comment:
this triggered now unhandled output in test output, e.g. see https://github.com/os-autoinst/os-autoinst/actions/runs/12260500123/job/34205258725#step:3:526

I opened https://progress.opensuse.org/issues/174094 . Should be easy to cover.